test/python/circuit/test_circuit_properties.py

```diff
@@ -17,15 +17,30 @@
 """Test Qiskit's inverse gate operation."""
 
 import unittest
+import numpy as np
 from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
 from qiskit.test import QiskitTestCase
+from qiskit.circuit.exceptions import CircuitError
 # pylint: disable=unused-import
 from qiskit.extensions.simulator import snapshot
 
 
 class TestCircuitProperties(QiskitTestCase):
     """QuantumCircuit properties tests."""
 
+    def test_qarg_numpy_int(self):
+        """Test castable to integer args for QuantumCircuit.
+        """
+        n = np.int64(12)
+        qc1 = QuantumCircuit(n)
+        self.assertEqual(qc1.n_qubits, 12)
+        self.assertEqual(type(qc1), QuantumCircuit)
```

Comment:
do you mind also adding a test for `int64` in the classical argument?

Reply:
Sorry, not sure what you mean exactly. Do you mean write a test creating a classical register using int64 and passing the classical register into the quantum circuit?

Reply:
Yes, something like `QuantumCircuit(12, np.int64(12))`.